ui/src/turing/components/form/standard_ensembler/LinkedExperimentsContextMenu.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+
+import {
+  EuiContextMenu,
+  EuiPopover,
+  EuiButtonEmpty,
+  EuiTextColor,
+  EuiIcon
+} from "@elastic/eui";
+
+export const LinkedExperimentsContextMenu = ({
+  item,
+  projectId,
+  routeToExperimentMappings,
+  isButtonPopoverOpen,
+  setIsButtonPopoverOpen,
+  experimentStatus
+}) => {
+  let numRoutes = routeToExperimentMappings[item.id] ? Object.keys(routeToExperimentMappings[item.id][experimentStatus]).length : 0;
+
+  const onButtonClick = () => {
+    let newIsButtonPopoverOpen = { ...isButtonPopoverOpen };
+    newIsButtonPopoverOpen[item.id][experimentStatus] = !isButtonPopoverOpen[item.id][experimentStatus];
+    setIsButtonPopoverOpen(newIsButtonPopoverOpen);
+  };
+
+  const closePopover = () => {
+    let newIsButtonPopoverOpen = { ...isButtonPopoverOpen };
+    newIsButtonPopoverOpen[item.id][experimentStatus] = false;
+    setIsButtonPopoverOpen(newIsButtonPopoverOpen);
+  };
+
+  const button = (
+    <EuiButtonEmpty
+      size={"s"}
+      iconType={"arrowRight"}
+      iconSide={"right"}
+      color={"primary"}
+      onClick={onButtonClick}
+      isDisabled={numRoutes === 0}
+    >
+      {numRoutes}
+    </EuiButtonEmpty>
+  );
+
+  return isButtonPopoverOpen[item.id] ? (
```

Comment:
Let's use `&&` and not have a ternary that returns `null` here.

Reply:
Thanks for catching that! I've just edited it!

Reply:
Correction: I remembered I need `null` as a return value because a function passed to the `render` prop needs to return minimally a `null` response as opposed to `undefined` which is what may happen if you use `&&` 
![Screenshot 2022-09-23 at 4 20 55 PM](https://user-images.githubusercontent.com/36802364/191919786-2bdcb6ab-f3ba-4ab0-957d-d3561acafaad.png)

Reply:
Should be addressed by this suggestion - https://github.com/caraml-dev/xp/pull/37#discussion_r979531215.